server/ocfg/config.go

```diff
@@ -0,0 +1,163 @@
+package ocfg
```

Comment:
```suggestion
package envconfig
```

or `env` might help users by telling them where the values come from